apps/mobile/src/components/ChatDetailScreen/components/VexlbotMessageItem/components/TradeChecklistAmountView.tsx

```diff
@@ -76,19 +75,19 @@ function TradeChecklistAmountView(): JSX.Element | null {
     [amountDataToDisplay, btcPriceForTradeCurrency?.btcPrice]
   )
 
-  const btcAmount = useMemo(
+  const fiatAmount = useMemo(
     () =>
-      amountDataToDisplay?.amountData.btcAmount
-        ? formatBtcPrice(
-            applyFeeOnBtcAmount(
-              amountDataToDisplay.amountData.btcAmount,
+      amountDataToDisplay?.amountData.fiatAmount
```

Comment:
So the value in amountDataToDispaly.amountData.fiatAmount is not actually the amount that is displayed right? Would it be possible to move the memo logic inside the atom?